.travis.yml

```diff
@@ -3,12 +3,7 @@ jdk:
   - openjdk7
 os:
   - linux
-cache:
-  directories:
-    - ~/.m2/repository
 install:
-  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true --quiet --batch-mode --show-version --activate-profiles travis
-before_script:
-  - echo "MAVEN_OPTS=${TEST_MEM_SETTINGS}" > ~/.mavenrc
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -P travis
```

Comment:
I prefer the long version of the params - --activate-profiles is clearer than -P so please put these back to how they were.

Reply:
Changed back to how it was